client/Lykke.Service.BlockchainCashoutPreconditionsCheck.Client/AutorestClient/BlockchainCashoutPreconditionsCheckAPI.cs

```diff
@@ -1225,7 +1225,10 @@ public async Task<HttpOperationResponse<object>> CheckWithHttpMessagesAsync(stri
             {
                 _queryParameters.Add(string.Format("AssetId={0}", System.Uri.EscapeDataString(assetId)));
             }
-            _queryParameters.Add(string.Format("Amount={0}", System.Uri.EscapeDataString(SafeJsonConvert.SerializeObject(amount, SerializationSettings).Trim('"'))));
+            if (amount != null)
```

Comment:
Looks like you changed autogenerated files - it's bad practice - cause you changed could be easily lost.